db/structure.sql

```diff
@@ -1541,5 +1812,7 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20200621214426'),
 ('20200621222659'),
 ('20200621223318');
+('20200629173821'),
```

Comment:
This is a bad merge it seems, the semicolon in the line above will not let these 2 additional versions be valid

Reply:
Fixed. Thank you